### ISSUE TITLE
PsiReturnStatementImpl are in unnamed module of loader 

### ISSUE THREAD

**Issue by arturmkr** [Legionivo/Allure-to-TestRail-exporter#4]
Having following test:
```
String token;

@Epic(value = "11")
  @Feature(value = "22")
  @Story(value = "333")
  @DisplayName(value = "something")
  @Test
  void someThingTest() {
    String val = getAuthorizationToken();
  }
```

This code doesn't work:
```
  @Step("Get auth token")
  protected String getAuthorizationToken() {
    if (token==null){
    token = tokenBase.getAuthToken();
    }
    return token;
  }
```

This code works:
```
  @Step("Get auth token")
  protected String getAuthorizationToken() {
    return tokenBase.getAuthToken();
  }
```

Following error in IDEA:
`  java.lang.ClassCastException: class com.intellij.psi.impl.source.tree.java.PsiExpressionStatementImpl cannot be cast to class com.intellij.psi.impl.source.tree.java.PsiReturnStatementImpl (com.intellij.psi.impl.source.tree.java.PsiExpressionStatementImpl and com.intellij.psi.impl.source.tree.java.PsiReturnStatementImpl are in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @6926d031)`

**Comment by Legionivo** [comment]
Hi there, @arturmkr 
I can't reproduce your issue, could you please provide more details? 

![Screenshot 2021-07-17 at 19 44 57](https://user-images.githubusercontent.com/12167296/126045549-81056501-f367-4dcb-9f71-ca5f343bf531.png)

**Comment by arturmkr** [comment]
@Legionivo 

Please check package `"mthIssue"`:
https://github.com/arturmkr/testrailToAllure/tree/main/src/test/java/com/example/TestRail/mthIssue

someTest1 - can't be exported
someTest2 - can be exported

**Comment by Legionivo** [comment]
@arturmkr fixed in v 1.6.0, please wait for an update from Jetbrains plugin repository
